eq-author-api/README.md

```diff
@@ -1,17 +1,37 @@
-# eq-author-api
+# eq-author- pi
 
 A GraphQL based API for the [eq-author](https://github.com/ONSdigital/eq-author)
 application.
 
 ## Table of Contents
 
-- [Installation](#installation)
-- [Environment Variables](#environment-variables)
-- [Run using Docker](#run-using-docker)
-- [Tests](#tests)
-- [Migrations](#migrations)
-- [Debugging (with VS Code)](#debugging-with-vs-code)
-- [Importing Questionnaires)](#importing-questionnaires)
+- [eq-author- pi](#eq-author--pi)
```

Comment:
This looks odd.

Reply:
It was changed by a mistake, have fixed it now.